src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
```

Comment:
Font-style, font-weight, font-size and line-height are different in different blocks on the site. These properties are best set in blocks

Reply:
these are styles for all page. They are used in header, and maybe will be used in another places on the page. Styles for card block I separeted in card styles.